projects/pipelines/testing/test.py

```diff
@@ -215,12 +223,11 @@ def prepare_cluster():
     # Update the MAX_CONCURRENT_RECONCILES if needed
     max_concurrent_reconciles = config.ci_artifacts.get_config("tests.pipelines.max_concurrent_reconciles")
     if max_concurrent_reconciles is not None:
-        updated_param = {
-            "data": {
-                "MAX_CONCURRENT_RECONCILES": str(max_concurrent_reconciles)
-            }
-        }
-        run.run(f"oc patch cm data-science-pipelines-operator-dspo-parameters -n redhat-ods-applications --patch '{json.dumps(updated_param)}'")
+        rhoai_down() # Pause RHOAI so we can edit a variable in the DSPO
+        run.run(f"oc set env deployment/data-science-pipelines-operator-controller-manager MAX_CONCURRENT_RECONCILES={str(max_concurrent_reconciles)} -n redhat-ods-applications")
+        # This is less than ideal, but I couldn't find a succinct way to wait until the pods
+        # from this deployment have all succesfully been redeployed
+        time.sleep(30)
```

Comment:
@kpouget I don't love this solution, but while looking over the deployment and configmap resources I wasn't able to notice a particular condition to wait on

Reply:
```
oc set env deployment/data-science-pipelines-operator-controller-manager
```
you update a deployement here, so you can wait ... for the deployment to be ready!

Reply:
Hmmmm ok, let me test that again tomorrow. When I tested it manually before, I don't think it ever left `Ready`. The deployment just slowly rolled out new pods and took down the old.

Reply:
hum, good point. I'm not very familiar with deployment *update*, but there are some other fields in the status that you can track, like the number not available, number ready